Dockerfile

```diff
@@ -127,41 +93,37 @@ RUN pip3 install --requirement=/nextstrain/fauna/requirements.txt
 # accessible and importable.
 RUN pip3 install --editable /nextstrain/augur
 
+# Install additional "full" dependencies for augur.
+# TODO: these versions should be updated in augur's setup.py to work with the
+# pip install nextstrain-augur[full] approach.
+RUN pip3 install cvxopt==1.2.4
+RUN pip3 install matplotlib==2.2.2
+RUN pip3 install seaborn==0.9.0
+
 # Install Node deps, build Auspice, and link it into the global search path.  A
 # fresh install is only ~40 seconds, so we're not worrying about caching these
 # as we did the Python deps.  Building auspice means we can run it without
 # hot-reloading, which is time-consuming and generally unnecessary in the
 # container image.  Linking is equivalent to an editable Python install and
 # used for the same reasons described above.
-RUN cd /nextstrain/auspice && npm install && npm run build && npm link
-
+RUN cd /nextstrain/auspice && npm update && npm install && npm run build && npm link
```

Comment:
@huddlej I was trying to answer "Why do we run `npm update`?" (when working on #165) and traced its introduction back to here. I assume it's unlikely, but any recollection why you added it?

Reply:
@tsibley I'm sorry I don't remember. Since I know very little about the npm ecosystem, it seems likely that I saw this pattern somewhere during work on this PR and copied it while trying to debug build issues.

Reply:
No worries!